src/main/java/com/yo1000/sonarqube/collaboration/Application.java

```diff
@@ -5,6 +5,14 @@
  */
 public class Application {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://dev.yo1000.com/coding_rules#rule_key=squid%3AS1118)